docs/root/operations/admin.rst

```diff
@@ -770,11 +770,11 @@ modify different aspects of the server:
     Text readout stats create a new label value every time the value
     of the text readout stat changes, which could create an unbounded number of time series.
 
-  .. http:get:: /stats?format=prometheus&histogram_emit_mode=histogram,summary
+  .. http:get:: /stats?format=prometheus&histogram_buckets=none
```

Comment:
For text and json, all 4 histogram_buckets modes are supported, and currently for prom, that is ignored, and in this PR you are adding support for 'summary'.

Right now you don't have 'detailed' or 'cumulative' supported for Prom and I'd consider that tech-debt; we don't have to solve it in this PR but we should leave TODOs. Actually TBH I'm not sure why 'cumulative' is ever useful but someone wanted it in the past and added it.

And we should document exactly what happens if the user specifies a not-yet-supported option like prometheus/detailed or prometheus/cumulative.

WDYT?

Reply:
The prometheus exposition format defines exactly how histograms are expressed. The only configurable/changeable behavior is the published buckets. No other options should be supported on prometheus histograms. (With the exception of switching to summaries instead of histograms; that is also well-defined in the exposition format)